tests/relay/test_node.py

```diff
@@ -0,0 +1,291 @@
+from typing import Any, List, Optional
+
+import pytest
+import strawberry
+from sqlalchemy import Column, Integer, String
+from sqlalchemy.engine import Engine
+from sqlalchemy.ext.asyncio.engine import AsyncEngine
+from sqlalchemy.orm import sessionmaker
+from strawberry import relay
+from strawberry_sqlalchemy_mapper import StrawberrySQLAlchemyMapper, node
+
+
+@pytest.fixture
+def fruit_table(base: Any):
+    class Fruit(base):
+        __tablename__ = "fruit"
+        id = Column(Integer, autoincrement=True, primary_key=True)
+        name = Column(String(50), nullable=False)
+        color = Column(String(50), nullable=False)
+
+    return Fruit
+
+
+def test_node(
+    base: Any,
+    engine: Engine,
+    sessionmaker: sessionmaker,
+    fruit_table,
+):
+    base.metadata.create_all(engine)
+    mapper = StrawberrySQLAlchemyMapper()
+
+    @mapper.type(fruit_table)
+    class Fruit(relay.Node):
```

Comment:
Do you think there's any way to create this type by default in the library through primary key abstraction? Doesn't have to be in this PR, but I really would love us to get to zero boilerplate

Reply:
Probably there is a way, indeed! It goes beyond the scope of this PR, but it can be done in the future

Reply:
There definitely is a way, but the zero boilerplate should be optional. Explicitly selecting what to include in the schema on a type level has advantages too, so I'm all for supporting both ways.